v0/destinations/marketo/util.js

```diff
@@ -77,7 +78,13 @@ const marketoApplicationErrorHandler = (
   }
 };
 
-const marketoResponseHandler = (destResponse, sourceMessage, stage) => {
+const marketoResponseHandler = (
+  destResponse,
+  sourceMessage,
+  stage,
+  destInfo,
```

Comment:
We should call it `destReqMetadata` or simply `reqMetadata`.

Reply:
It is `destInfo` that is being sent from `responseHandler` function. I don't think there is a need to change the name

Reply:
I don't want to debate further on the naming, but in general, it's metadata IMO, which should only be destructured at the last node of usage.

Reply:
> which should only be destructured at the last node of usage.

Yes. As I found out about `marketoResponseHandler` being used for parsing intermediate API calls. @ujjwal-ab can you make relevant changes ?

Reply:
We will have to destructure the metadata in the `responseHandler` itself as we are using the same `marketoResponseHandler` function for cache eviction for internal lookup call failure and proxy both so we will have to send the same params in both cases as we are passing the `authCache` and `authKey` in case of internal lookups.

Reply:
Yeah, you can wrap up that data in other places accordingly. The idea is to stick to a generic format for passing the data across the layers.

Reply:
Should we make the internal lookups format in the proxy format?

Reply:
Yes, you can wrap it under the veil of metadata.